src/components/MdDatepicker/MdDatepicker.vue

```diff
@@ -85,7 +85,7 @@
           const parsedDate = parse(value)
 
           if (isValid(parsedDate)) {
-            this.selectedDate = parsedDate
+            this.selectedDate = this.dateToHTMLString(parsedDate)
```

Comment:
Still got error here:

* click the datepicker input field
* select any date <= 12
* click *OK* button
* click the datepicker input field again
* got `[Vue warn]: Invalid prop: type check failed for prop "mdDate". Expected Date, got String.`

Reply:
![apr-25-2018 23-42-09](https://user-images.githubusercontent.com/29639463/39256824-792fec14-48e2-11e8-8657-b88b8bae0e98.gif)